### PR TITLE
Update MangaWorld presence

### PR DESCRIPTION
**Pull request by TheTecnoKing**
Remade the MangaWorld presence to support their new website.
### Screenshots:
<details>
<summary>Homepage</summary>

homepage
</details>

<details>
<summary>Bookmarks</summary>

*   Main bookmarks
*   Reading bookmarks_reading
*   Completed bookmarks_completed
*   Paused bookmarks_paused
*   Plan to read bookmarks_plantoread
*   Dropped bookmarks_dropped
*   Re-reading bookmarks_rereading
</details>

<details>
<summary>Archive</summary>

*   Main archive
*   Search archive_search
*   Sort by artist archive_artist
*   Sort by author archive_author
*   Sort by genre archive_genre
*   Sort by type archive_type
*   Sort by status archive_status
*   Sort by year archive_year
*   Order by archive_orderby
</details>

<details>
<summary>Reading</summary>

*   Main reading
*   In the e-reader reading_ereader *Broken embed due to "Content length exceeded", please click [this](https://i.imgur.com/UtSyJi7.gif) instead.*
*   "Read along!" button reading_readalong
</details>

<details>
<summary>Other pages</summary>

otherpages
</details>

### CODE REVIEW

File: websites/M/MangaWorld/dist/metadata.json [hunk 1]
```diff
@@ -3,24 +3,30 @@
...
-    "mangaworld.tv",
-    "www.mangaworld.tv"
+    "mangaworld.io",
+    "www.mangaworld.io"
   ],
-  "version": "2.1.9",
-  "logo": "https://i.imgur.com/MdYzAHr.png",
-  "thumbnail": "https://i.imgur.com/BFsORve.png",
-  "color": "#EE6B00",
+  "version": "3.0.6",
```
Comment (Bas950) [inline websites/M/MangaWorld/dist/metadata.json:22]:
```suggestion
  "version": "3.0.0",
```

**Review by Bas950 — CHANGES_REQUESTED** [review]
Fix errors reported by the bots

**Comment by TheTecnoKing** [comment]
Do I have to fix all the indentation and curly brace warnings as well?
I left the braces not on the same line with the next block on purpose, so I could write comments for each section.

**Comment by Bas950** [comment]
> Do I have to fix all the indentation and curly brace warnings as well?
> I left the braces not on the same line with the next block on purpose, so I could write comments for each section.
> @Bas950

Yes, please use our prettier settings etc.
You can also put the comment in the first line after the if case

File: websites/M/MangaWorld/presence.ts [hunk 2]
```diff
@@ -1,193 +1,201 @@
...
-    data.state = document.title.split("Archivi - ")[0];
+  } else {
+    /* In qualunque altra pagina - In any other page */
+    const pageName = document.title.replace("MangaWorld - ", "");
+    data.smallImageKey = "eye";
+    data.smallImageText = `"${pageName}"`;
+    data.details = "Visualizzando la pagina:";
+    data.state = `"${pageName}"`;
+  }
+  if (data.details !== null) {
```
Comment (Bas950) [inline websites/M/MangaWorld/presence.ts:198]:
```suggestion
  if (data.details) {
```

**Review by Slowlife01 — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**